ehrapy/_settings.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         figdir: str | Path = "./figures/",
         cache_compression: str | None = "lzf",
         max_memory=15,
-        n_jobs: int = 1,
```

Comment:
Can this have more effects than we hope for..? I vaguely recall a method in a tutorial once having failed with n_jobs=-1, but not sure if really was this or something joblib related (it was a bug in a package though).

Not sure if it would be safer to keep n_jobs as 1, users can still set this to -1 at their wish

Reply:
It should be good I think.